crates/topos/Cargo.toml

```diff
@@ -15,6 +15,7 @@ topos-core = { workspace = true, features = ["api"] }
 topos-certificate-spammer = { path = "../topos-certificate-spammer", optional = true }
 topos-tce-broadcast = { path = "../topos-tce-broadcast", optional = true }
 topos-wallet = { path = "../topos-wallet" }
+topos-test-sdk = { path = "../topos-test-sdk", features = ["tce"] }
```

Comment:
So we have to have this built even for non-test builds?

Reply:
No I forgot to remove it